xcdat/bounds.py

```diff
@@ -1037,6 +1043,14 @@ def get_bounds_dim(da_bounds: xr.DataArray) -> str:
     This function is designed to work with DataArrays that include a dimension
     representing bounds, which is commonly used in geospatial or temporal data.
     """
+    # If only one bounds dimension is found, return it.
+    bounds_dim = [dim for dim in da_bounds.dims if dim not in da_coords.dims]
+
+    if len(bounds_dim) == 1:
+        return str(bounds_dim[0])
+
+    # FIXME: Do we still need this logic, or do we always expect bounds to have
+    # one additional dimension?
     for dim in VALID_BOUNDS_DIMS:
         if dim in da_bounds.dims:
```

Comment:
Hey @pochedls, do you recall if there is ever a case where coordinate bounds have >1 dimension more than their coordinates?

For example:

```python
time: (time)
time_bnds: (time, bnds, some_other_dim)
```

If not, then we can remove the old-logic with the hard-coded list.

Refer to [PR description](https://github.com/xCDAT/xcdat/pull/773#issue-3131611627) on how this works now.

Reply:
From my reasoning, bounds should only have 1 more dimension than their coordinates.

For example
```python
time: (time)
time_bnds: (time, nv)

Result
------
"nv" is extracted 
```

```python
time: (member, time)
time_bnds: (member, time, bnds)

Result
--------
"bnds" is extracted
```

Reply:
@tomvothecoder – sorry for the slow response. I don't know how I missed your comment three weeks ago. I did not understand the comment/question, but I now do. I think this logic makes sense. I haven't seen a `bound` dim in `coords` before, but I wonder if it does happen sometimes?